compiler/src/scala/quoted/runtime/impl/printers/SourceCode.scala

```diff
@@ -809,13 +809,10 @@ object SourceCode {
         case Nil => Flags.EmptyFlags
         case arg :: _ => arg.symbol.flags
       }
-      if (argFlags.is(Flags.Erased | Flags.Given)) {
-        if (argFlags.is(Flags.Given)) this += " given"
-        if (argFlags.is(Flags.Erased)) this += " erased"
-        this += " "
-      }
       inParens {
         if (argFlags.is(Flags.Implicit) && !argFlags.is(Flags.Given)) this += "implicit "
+        if (argFlags.is(Flags.Given)) this += "using "
+        if (argFlags.is(Flags.Erased)) this += "erased "
```

Comment:
This one is not correct, `erased` can appear on each individual parameter.

Reply:
I'm hoping 7f9208b58127047c0197d60077bde9b53decad44 fixed this. Do let me know if you think it's not in the right place.